src/js/visual/configurator-csr.js

```diff
@@ -1363,3 +1544,19 @@ ripe.ConfiguratorCsr.prototype._unregisterInitialsHandlers = function() {
     this.owner && this.owner.unbind("initials_extra", this._onInitialsExtraEvent);
     this.owner && this.owner.unbind("initials", this._onInitialsEvent);
 };
+
+/**
+ * @ignore
+ */
+ripe.ConfiguratorCsr.prototype._registerConfigHandlers = function() {
+    this.owner.bind("pre_config", () => this._onPreConfig(this));

```

Comment:
can we keep this consistent like this, even if we're not using?
```suggestion
    this.owner.bind("pre_config", config => this._onPreConfig(this, config));

```

Reply:
Yes!
Altough it would be `brand, model, options` 👍

Reply:
nevermind then!